pkg/authorization/api/types.go

```diff
@@ -72,7 +72,7 @@ var (
 		UserGroupName:               {"identities", "users", "useridentitymappings", "groups"},
 		OAuthGroupName:              {"oauthauthorizetokens", "oauthaccesstokens", "oauthclients", "oauthclientauthorizations"},
 		PolicyOwnerGroupName:        {"policies", "policybindings"},
-		PermissionGrantingGroupName: {"roles", "rolebindings", "resourceaccessreviews", "subjectaccessreviews"},
+		PermissionGrantingGroupName: {"roles", "rolebindings", "resourceaccessreviews" /* cluster scoped*/, "subjectaccessreviews" /* cluster scoped*/, "localresourceaccessreviews", "localsubjectaccessreviews"},
```

Comment:
are the cluster-scoped resources only included for backwards compatibility (the other cluster-level objects like clusterroles aren't in this list)? if so, add a comment to remove them once support for /namespaces/ns/{subjectaccessreview,resourceaccessreview} is removed